lib/index.js

```diff
@@ -156,6 +156,50 @@ function createIntegerConversion(bitLength, typeOpts) {
     };
 }
 
+function createLongLongConversion(bitLength, { unsigned }) {
+    const upperBound = Number.MAX_SAFE_INTEGER;
+    const lowerBound = unsigned ? 0 : Number.MIN_SAFE_INTEGER;
+    const asBigIntN = unsigned ? BigInt.asUintN : BigInt.asIntN;
+
+    return (V, opts = {}) => {
+        if (opts === undefined) {
+            opts = {};
+        }
+
+        let x = toNumber(V, opts);
+        x = censorNegativeZero(x);
+
+        if (opts.enforceRange) {
+            if (!Number.isFinite(x)) {
+                throw makeException(TypeError, "is not a finite number", opts);
+            }
+
+            x = integerPart(x);
+
+            if (x < lowerBound || x > upperBound) {
+                throw makeException(TypeError,
+                    `is outside the accepted range of ${lowerBound} to ${upperBound}, inclusive`, opts);
+            }
+
+            return x;
+        }
+
+        if (!Number.isNaN(x) && opts.clamp) {
+            x = Math.min(Math.max(x, lowerBound), upperBound);
+            x = evenRound(x);
+            return x;
+        }
+
+        if (!Number.isFinite(x) || x === 0) {
+            return 0;
+        }
+
+        let xBigInt = BigInt(integerPart(x));
```

Comment:
Note: this no longer uses `BigInt(Number(integerPart(x)))`, as `toString` causes some precision to be lost. See the note at https://tc39.es/ecma262/#sec-number.prototype.tofixed:

> The output of toFixed may be more precise than toString for some values because toString only prints enough
> significant digits to distinguish the number from adjacent number values. For example,
>
> `(1000000000000000128).toString()` returns "1000000000000000100", while `(1000000000000000128).toFixed(0)` returns "1000000000000000128".